deploy/charts/cert-manager/values.yaml

```diff
@@ -39,6 +39,10 @@ global:
     # renewal of a leadership.
     # retryPeriod: 15s
 
+# Setting this value to "helm" or "static"
+# will determine what labels to add to each resource
+creator: "helm"
```

Comment:
This is the bit I disagree with Tim.
This implementation detail will now be exposed to our users as a confusing new Helm template variable, 
which with #4199 will also show up in the chart README file.

Isn't there some builtin variable in one of the other Helm namespaces that we could use to tell whether the user called `helm template` or `helm upgrade / install`, and only in the latter case would we add the  `helm.sh/chart` annotation.

What is the value of `.Release.Service` if `helm template` is used?

https://v3-1-0.helm.sh/docs/chart_best_practices/labels/

Reply:
I'll remove the creator parameter from the `values.yaml` file.